tsconfig.build.json

```diff
@@ -0,0 +1,35 @@
+{
+  "include": ["src/**/*"],
+  "compilerOptions": {
+    "module": "commonjs",
+    "strict": true,
+    "declaration": true,
+    "noImplicitAny": true,
+    "removeComments": true,
+    "noLib": false,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "esModuleInterop": true,
+    "target": "es2018",
+    "sourceMap": true,
+    "lib": ["es2018", "dom"],
+    "moduleResolution": "node",
+    "outDir": "./dist"
+  },
+  "exclude": [
+    "../../node_modules", "dist", "**/__mocks__/**"],
```

Comment:
Yeah, I'm not sure what the intent of this `exclude` was originally, so I'm not sure how to update it for Lerna - I mean what should the 3 values here be updated to, e.g. `"packages/*/../../node_modules", "packages/*/dist", "packages/*/**/__mocks__/**"` ... ???

Reply:
https://www.typescriptlang.org/tsconfig#exclude `exclude` aims at removing some files from the transpiling step if they are included in the `include` glob. Something like `**/node_modules/, **/__mocks__/, **/dist/` looks like it could work.

Reply:
Ok - done.